get_sig_pitch.m

```diff
@@ -85,6 +99,12 @@
     
 
     otherwise
```

Comment:
```suggestion
    otherwise % if ptrack_method is anything other than praat, use legacy pitch tracking code
```

Reply:
Fixed in d759fd9e04ac8570381d8984284acff8925938fd